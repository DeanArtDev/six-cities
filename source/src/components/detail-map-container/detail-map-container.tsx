import * as React from 'react';

export const WrappedComponent = React.forwardRef<HTMLElement, React.HTMLProps<HTMLElement>>(
    function CitiesMapContainer(props, ref) {
      return (
        <section className="property__map map" id="map" ref={ref}/>
      );
    }
);

