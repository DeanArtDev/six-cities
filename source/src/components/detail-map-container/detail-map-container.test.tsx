import {WrappedComponent as DetailMapContainer} from './detail-map-container';
import * as renderer from 'react-test-renderer';
import * as React from 'react';

describe(`<DetailMapContainer/>: `, () => {

  it(`should render correctly`, () => {
    const tree = renderer
      .create(
          <DetailMapContainer/>
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
