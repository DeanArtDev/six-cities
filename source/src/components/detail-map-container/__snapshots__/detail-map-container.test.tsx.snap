// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DetailMapContainer/>:  should render correctly 1`] = `
<section
  className="property__map map"
  id="map"
/>
`;
